Week-3/solution-2.js

```diff
@@ -0,0 +1,55 @@
+//Solution-2
+
+//call, apply, bind
+
+//bind
+
+const obj1 = {
+  name: "Sachiv",
+  place: "Bihar",
+  job: function work(){
```

Comment:
Is the function name required, 
As the key is already defined. So, do we need the work as a function name?